UserShiftsApiService/UserShiftsApiService/Controllers/UserController.cs

```diff
@@ -0,0 +1,31 @@
+using System.Security.Claims;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using UserShiftsApiService.Models;
+
+namespace UserShiftsApiService.Controllers;
+
+[ApiController]
+[Route("/users/")]
+public class UserController : ControllerBase
+{
+    private readonly ShiftsSchedulingContext _dbContext;
+
+    public UserController(ShiftsSchedulingContext dbContext)
+    {
+        _dbContext = dbContext;
+    }
+    
+    [HttpGet]
+    [Authorize]
+    [Route("get-user")]
+    public async Task<IActionResult> GetUser()
+    {
+        var authSub = User.FindFirst(ClaimTypes.NameIdentifier)?.Value;
```

Comment:
Lets make it so u cant even get here without a userId being loaded from DB (use authorization middleware again)